### ISSUE TITLE
Tabular view error

### ISSUE THREAD

**Issue by petschki** [plone/plone.app.standardtiles#122]
When you choose "Display mode" -> "Tabular view" there's an exception:

```
Traceback (innermost last):
  Module ZPublisher.WSGIPublisher, line 167, in transaction_pubevents
  Module ZPublisher.WSGIPublisher, line 376, in publish_module
  Module ZPublisher.WSGIPublisher, line 271, in publish
  Module ZPublisher.mapply, line 85, in mapply
  Module ZPublisher.WSGIPublisher, line 68, in call_object
  Module plone.app.standardtiles.contentlisting, line 162, in __call__
  Module Products.Five.browser.pagetemplatefile, line 126, in __call__
  Module Products.Five.browser.pagetemplatefile, line 58, in __call__
  Module zope.pagetemplate.pagetemplate, line 133, in pt_render
  Module Products.PageTemplates.engine, line 365, in __call__
  Module z3c.pt.pagetemplate, line 176, in render
  Module chameleon.zpt.template, line 302, in render
  Module chameleon.template, line 192, in render
  Module f9913b725e41023d075b506f446dc8ad, line 264, in render
  Module zope.tales.pythonexpr, line 73, in __call__
   - __traceback_info__: (view.contents())
  Module <string>, line 1, in <module>
  Module plone.app.standardtiles.contentlisting, line 250, in contents
  Module zope.browserpage.simpleviewclass, line 41, in __call__
  Module Products.Five.browser.pagetemplatefile, line 126, in __call__
  Module Products.Five.browser.pagetemplatefile, line 58, in __call__
  Module zope.pagetemplate.pagetemplate, line 133, in pt_render
  Module Products.PageTemplates.engine, line 365, in __call__
  Module z3c.pt.pagetemplate, line 176, in render
  Module chameleon.zpt.template, line 302, in render
  Module chameleon.template, line 214, in render
  Module chameleon.utils, line 53, in raise_with_traceback
  Module chameleon.template, line 192, in render
  Module e91c30ff11bb8410c8aa383dcabb966e, line 208, in render
  Module zope.tales.expressions, line 326, in __call__
  Module Products.PageTemplates.Expressions, line 246, in evaluateBoolean
  Module zope.tales.tales, line 775, in evaluate
   - Expression: <PathExpr standard:'results'>
   - Names:
      {'args': (),
       'container': <plone.batching.batch.BaseBatch object at 0x114b4b100>,
       'context': <plone.batching.batch.BaseBatch object at 0x114b4b100>,
       'default': <DEFAULT>,
       'here': <plone.batching.batch.BaseBatch object at 0x114b4b100>,
       'loop': {},
       'nothing': None,
       'options': {'original_context': <Collection at /Plone/filterbar>},
       'repeat': <Products.PageTemplates.Expressions.SafeMapping object at 0x1149973c0>,
       'request': <WSGIRequest, URL=http://localhost:8080/Plone/filterbar/@@plone.app.standardtiles.contentlisting/0009f65cbddc4e08aa72d1e786f07fad>,
       'root': None,
       'template': <Products.Five.browser.pagetemplatefile.ViewPageTemplateFile object at 0x114b4bee0>,
       'traverse_subpath': [],
       'user': <PropertiedUser 'admin'>,
       'view': <Products.Five.browser.metaconfigure.SimpleViewClass from /Volumes/WORKSPACE2/spielboden-6.0/eggs/plone.app.standardtiles-2.4.0-py3.8.egg/plone/app/standardtiles/templates/tabular_view.pt object at 0x114b4bf70>,
       'views': <Products.Five.browser.pagetemplatefile.ViewMapper object at 0x114b2ea60>}
  Module zope.tales.expressions, line 250, in __call__
  Module Products.PageTemplates.Expressions, line 184, in _eval
  Module zope.tales.expressions, line 143, in _eval
  Module Products.PageTemplates.engine, line 176, in __getitem__
KeyError: 'results'
```

**Comment by mauritsvanrees** [comment]
Fixed directly on master. Released as 3.0.0a2.

**Comment by mauritsvanrees** [comment]
Also released in 2.4.2.
